server/controllers/userController.js

```diff
@@ -321,6 +326,64 @@ class UserController {
       util.errorStatus(res, 500, error.name);
     }
   }
+
+  static async getProfile(req, res) {
+    const { is_own_profile } = req;
```

Comment:
What did you use this for?

Reply:
I imagined how a front end person would know when the user is trying to fetch his or another person's profile, so i added that to the response. With this, the front end person would know how to dynamically  build up the profile pages. To be more specific, know when to add the edit button to a user profile

Reply:
But I cannot see where it was used?

Reply:
Oh I see. Can you stick to the camelCase convention

Reply:
Sure thing

Reply:
But then you can return the ID of the user and anyone can figure out whose profile it is

Reply:
Yeah, That is possible too. But that would mean I would be returning two ID's one for the visiting user, and one for the visited user. I just felt this would be easier for the front end person to implement.

Reply:
I'm here to learn more though, If you want me to change it,  I could easily.

Reply:
That's fine